client/src/Components/App.js

```diff
@@ -40,7 +52,8 @@ class App extends React.Component {
           </nav>
         </div>
         <div className="row">
-          <div className="col s12"><CreateAlert alert={this.state.alerts} /></div>
+          <div className="container"><CreateAlert alert={this.state.alerts[0]}  pairs={this.state.pairs}/></div>
```

Comment:
Are we always guaranteed to have at least one alert?
I'm trying to avoid any null exceptions with the `this.state.alerts[0]` statement